src/core/terms.ml

```diff
@@ -263,6 +263,34 @@ type ctxt = (term Bindlib.var * term * term option) list
 (** Type of a list of unification constraints. *)
 type unif_constrs = (ctxt * term * term) list
 
+(** Some definitions for unification hints. *)
+module Unif_hints = struct
+
+  (** Symbol representing an atomic unification problem. The term [atom t u]
+      represents [t =? u]. *)
+  let atom : term =
+    let sym =
+      { sym_name="unif_atom"; sym_type=ref Kind; sym_path=[]
+      ; sym_def=ref None; sym_impl=[];sym_tree=ref Tree_types.empty_dtree
+      ; sym_prop=Defin; sym_expo=Public; sym_rules=ref [] }
+    in
+    Symb(sym, Binary("≡"))
```

Comment:
It would be better that atom and list return symbols instead. Then, you won't need to_sym anymore. pp_hints are going to be removed soon anyway (see #363).